Assets/UI/Panels/citypanel.lua

```diff
@@ -630,47 +672,57 @@ function ViewMain( data:table )
         productionHash = currentProductionHash;
     end
     local currentProductionInfo :table = GetProductionInfoOfCity( data.City, productionHash );
+    -- ==== CQUI CUSTOMIZATION END ======================================================================================== --
```

Comment:
Git comparer is having a tough time with the extra comment lines here
the code itself did not change, only comment lines added to call out the CQUI changes from the unmodified file